base-template.sh

```diff
@@ -61,17 +62,22 @@ are_githooks_disabled() {
 # Returns: None
 #####################################################
 load_install_dir() {
-    INSTALL_DIR=$(git config --global --get githooks.installDir)
+
+    # INSTALL_DIR might be set from base-template-symlink.sh
```

Comment:
```suggestion
    # INSTALL_DIR might be passed from base-template-symlink.sh
```
I was a bit confused by the word "set", thinking it would actually set the variable directly (through environment or so).